webui/src/lib/remark-plugins/imageUriReplacer.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { remark } from "remark";
+import { beforeEach, describe, test, expect, vi, afterEach } from "vitest";
+import { objects } from "../api";
+
+import imageUriReplacer from "./imageUriReplacer";
+
+const TEST_REPO = "test-repo";
+const TEST_BRANCH = "test-branch";
+const TEST_FILE_NAME = "image.png";
+const ADDITIONAL_PATH = "additional/path";
+
+vi.mock("../api", () => {
+  const mockObjects = {
+    objects: {
+      getPresignedUrlForDownload: vi.fn(),
+    },
+  };
+  return mockObjects;
+});
+
+describe("imageUriReplacer", async () => {
+  let mockObjects: any;
```

Comment:
Notice the warning here

Reply:
Yes, I'm aware. Since it's a spec file and figuring out the TS support of vitest mocks was taking an unproportionate amount of time, I left it this way (at least for now)

Reply:
I removed it, so no more warning